src/services/print.js

```diff
@@ -280,12 +280,18 @@ ngeo.Print.prototype.encodeImageWmsLayer_ = function(arr, layer) {
  * @private
  */
 ngeo.Print.prototype.encodeWmsLayer_ = function(arr, opacity, url, params) {
+  const url_url = new URL(url);
   const customParams = {'TRANSPARENT': true};
-  // remove empty params
+  if (url_url.searchParams) {
+    for (const element of url_url.searchParams) {
+      customParams[element[0]] = element[1];
```

Comment:
I don't understand what you are doing here

Reply:
I convert the base URL query string in customParams (The first one is uppercased but not the second one ...)